commit 2f4ebeb2c27756c65906949dcf19663043089632
Author: Yorick Peterse <yorickpeterse@gmail.com>
Date:   Tue Jun 3 20:26:55 2014 +0200

    Implement RbConfig::CONFIG['NULLCMD']
    
    Based on some digging through the MRI source code this always seems to be set
    to ":".
    
    Fixes #3054.

diff --git a/library/rbconfig.rb b/library/rbconfig.rb
index 2e0cbba..d47cc43 100644
--- a/library/rbconfig.rb
+++ b/library/rbconfig.rb
@@ -106,6 +106,7 @@ module RbConfig
   CONFIG["STRIP"]              = "strip -A -n"
   CONFIG["MANTYPE"]            = "doc"
   CONFIG["MAKEFILES"]          = "Makefile"
+  CONFIG["NULLCMD"]            = ":"
   # compile tools flags
   # since we hardcode using gcc, and this flag is only
   # used by mkmf to compile extensions, be sure PIC is in
